src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -70,9 +70,31 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                 (token.Parent.IsKind(SyntaxKind.XmlElementEndTag) && token.IsKind(SyntaxKind.GreaterThanToken)) ||
                 (token.Parent.IsKind(SyntaxKind.XmlName) && token.Parent.IsParentKind(SyntaxKind.XmlEmptyElement)))
             {
+                // The user is typing inside an XmlElement
                 if (token.Parent.Parent.Kind() == SyntaxKind.XmlElement)
                 {
-                    items.AddRange(GetNestedTags(span));
+                    // Add corresponding tags except ParamRefTag and TypeParamRefTag
+                    items.AddRange(NestedTagNames.Where(name => name != ParamRefTagName && name != TypeParamRefTagName)
+                                                 .Select(name => GetItem(name, span)));
+
+                    // Add a ParamRefTag for each available parameter
+                    var symbol = declaredSymbol as IMethodSymbol;
+                    if (symbol != null)
+                    {
+                        var parameters = symbol.GetParameters().Select(p => p.Name).ToSet();
+                        items.AddRange(parameters.Select(p => CreateCompletionItem(span, FormatParameter(ParamRefTagName, p))));
+                    }
+
+                    // Add a TypeParamRefTag for each available type parameter
+                    var typeParameters = (declaredSymbol as INamedTypeSymbol)?.TypeParameters.Select(t => t.Name).ToSet();
```

Comment:
There's an equivalent helper for [getting type parameters](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Shared/Extensions/ISymbolExtensions.cs,12810581fa1bf76b)